docs/tools.md

```diff
@@ -1,96 +1,154 @@
 
 ## 服务器管理面板
 
-[1panel](https://1panel.cn/docs/)
+[1panel](https://1panel.cn/docs)
 
-[宝塔面板](https://www.bt.cn/new/product_Linux.html)
+[宝塔面板](https://www.bt.cn/new/download.html)
 
 ## 远程连接
 
 ### 串流
 
-[向日葵远程](https://sunlogin.oray.com/)
-
-[todesk](https://todesk.com/)
-
-[parsec](https://parsec.app/)
+[向日葵远程](https://sunlogin.oray.com)
 
-[moonlight](https://moonlight-stream.org/)
+[ToDesk](https://todesk.com)
 
-[Sunshine](https://github.com/LizardByte/Sunshine)
+[Parsec](https://parsec.app)
 
+[Moonlight](https://moonlight-stream.org)
 
+[Sunshine](https://app.lizardbyte.dev/Sunshine/?lng=zh-CN)
 
 ### Microsoft Remote Desktop
 
+[Windows](https://apps.microsoft.com/detail/9wzdncrfj3ps)
+
 [Mac OS](https://aka.ms/rdmacbeta)
 
 [Android](https://microsoft-corporation-com-microsoft-rdc-androidx.en.uptodown.com/android)
 
-[IOS](https://apps.apple.com/cn/app/microsoft-%E8%BF%9C%E7%A8%8B%E6%A1%8C%E9%9D%A2/id714464092)
+[IOS](https://apps.apple.com/cn/app/id714464092)
 
 ### VNC
 
-[realVNC](https://www.realvnc.com/)
+[RealVNC](https://www.realvnc.com)
 
-[tightvnc](https://www.tightvnc.com/download.php)
+[TightVNC](https://www.tightvnc.com/download.php)
 
 ### SSH
 
-[juicessh](https://juicessh.com/)
+[JuiceSSH](https://juicessh.com)
 
-[termius](https://www.termius.com/)
+[Termius](https://www.termius.com)
 
-[xshell](https://www.xshell.com/zh/xshell/)
+[Xshell](https://www.xshell.com/zh/xshell)
 
 ### 远程文件管理
 
-[winscp(仅Windows)](https://winscp.net/eng/download.php)
+[WinSCP (仅 Windows)](https://winscp.net/eng/download.php)
+
+[Forklift (仅 Mac OS)](https://binarynights.com)
 
-[Forklift(仅Mac OS)](https://binarynights.com/)
+[FileZilla Client (多平台)](https://filezilla-project.org/download.php)
 
 ## 游戏云相关依赖
 
+[Dragonwell JDK](https://dragonwell-jdk.io)
+
 [Microsoft Visual C++ Redistributable](https://learn.microsoft.com/zh-tw/cpp/Windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022)
 
-[DirectX修复工具](https://soft.3dmgame.com/down/148077.html)
+[DirectX 修复工具](https://soft.3dmgame.com/down/148077.html)
+
+[Xbox 手柄驱动](https://xbox-360-controller-for-Windows.en.lo4d.com/download)
+
+[远程虚拟手柄驱动](https://github.com/nefarius/ViGEmBus/releases/tag/setup-v1.17.333)
+
+## 常见解压缩工具
+
+[7-zip](https://www.7-zip.org)
+
+[Bandizip](https://cn.bandisoft.com)
+
+[WinRAR](https://www.rarlab.com/download.htm)
+
+## 常见串流播放器
+
+[VLC media player](https://www.videolan.org/index.zh_CN.html)
+
+[PotPlayer](https://potplayer.tv/?lang=zh_CN)
+
+[Kodi](https://kodi.tv)
+
+## 常见网络浏览器
+
+### Google Chrome
+
+[Google CN 下载](https://www.google.cn/chrome)
+
+[Google Play 下载](https://play.google.com/store/apps/details?id=com.android.chrome)
+
+[Apple Store 下载](https://apps.apple.com/cn/app/id535886823)
+
+### Microsoft Edge
+
+[Microsoft 下载](https://www.microsoft.com/zh-cn/edge/download)
+
+[Google Play 下载](https://play.google.com/store/apps/details?id=com.microsoft.emmx)
+
+[Apple Store 下载](https://apps.apple.com/cn/app/id1288723196)
```

Comment:
`https://apps.apple.com/cn/app/microsoft-edge-%E5%BE%AE%E8%BD%AF%E5%87%BA%E5%93%81%E6%B5%8F%E8%A7%88%E5%99%A8/id1288723196`